### ISSUE TITLE
Autocompletion on citations not working

### ISSUE THREAD

**Issue by brianichter** [SublimeText/LaTeXTools#1028]
When I use the \cite{} command, instead of opening the autocompletion dropbox for my potential citations, I get an error which states "An error occurred while trying to run kpsewhich. Files in your TEXINPUTS could not be accessed." 

It appears the error is output from here, https://github.com/SublimeText/LaTeXTools/blob/master/kpsewhich.py. I also looked at the LatexTools System Check and the definition for TEXINPUTS is quite long

Any ideas on how to fix it? Thanks!

Using ST3, OSX.

**Comment by r-stein** [comment]
Which command are you using to include your texfile? `kpsewhich` is only running, if it is not found in your document directory. (This is a misleading error message.)

**Comment by brianichter** [comment]
I'm using bibtex (I think that's what you're asking).

**Comment by r-stein** [comment]
You are including your bibfile via `\bibliography{references.bib}` and the file `references.bib` is inside the root folder?

**Comment by brianichter** [comment]
Your comment got me thinking and I realized the issue. I had a section below my \bibliography where I had some notes and \bibliography call to the same bib. Since I did not want to render the notes, I had them disabled by surrounding with /iffalse <notes> \fi. I think this redefined the bibliography I was looking for to the autocompleter, but since it was disabled during compilation it may have never actually gathered the bib information and couldn't access the file.

Thanks so much for the help!
